Chapters/6_Qualified_identifiers.md

```diff
@@ -25,6 +25,15 @@ designates the function `generate_intput_stream` that is found at lines 9 to 15
 Notice that the notion of "line number" is not always meaningful: the content
 may be a binary file, or a file that uses non standard line termination character(s).
 
+### 6.1.2 Bytes qualifier
```

Comment:
It looks like that, with the introduction of the bytes qualifier, for the first time we now have incompatible qualifiers, i.e., one can specify either bytes or lines (or none of them), but not *both* lines and bytes at the same time.
This is a big change in terms of (in)correctness.
I'm fine with it (because bytes is much better than lines anyway), but we should make it strongly explicit in the text.

In the section about lines qualifier we should say that one MUST not also use bytes with it; and vice-versa in the section about the bytes qualifier.

Reply:
Made this even more explicit in 25c2cb050c8893c9c413a261d6724696de4af536 Also added recommended behaviour on non conformant SWHIDs.